src/components/rocm/README.md

```diff
@@ -85,6 +85,21 @@ setting the ROCP\_TOOL\_LIB to the PAPI library as follows:
 
     The binary image of a `double` is intact; but users must recast to `double` for display purposes.
 
+* Some of the ROCm events are known to cause an error when the rocm component is used in sampling mode
+
+    For example TA_BUSY_avr
+
+    $ papi_command_line TA_BUSY_avr
+
+    This utility lets you add events from the command line interface to see if they work.
+
+    Successfully added: rocm:::TA_BUSY_avr:device=0
+
+    Memory access fault by GPU node-4 (Agent handle: 0x46d6d10) on address 0x7ffed888c000. Reason: Unknown.
+    Aborted
```

Comment:
I assume, this should be some input and output in a terminal, but thats not how it will render.
```suggestion
    ```console
    $ papi_command_line TA_BUSY_avr

    This utility lets you add events from the command line interface to see if they work.

    Successfully added: rocm:::TA_BUSY_avr:device=0

    Memory access fault by GPU node-4 (Agent handle: 0x46d6d10) on address 0x7ffed888c000. Reason: Unknown.
    Aborted
    ````
```